view/frontend/requirejs-config.js

```diff
@@ -20,13 +20,14 @@ var config = {
         }
     },
     paths : {
-        'algoliaBundle'         : 'Algolia_AlgoliaSearch/js/internals/algoliaBundle.min',
-        'algoliaSearchLib'      : 'Algolia_AlgoliaSearch/js/lib/algolia-search.min',
-        'algoliaAutocompleteLib': 'Algolia_AlgoliaSearch/js/lib/algolia-autocomplete.min',
-        'algoliaAnalytics'      : 'Algolia_AlgoliaSearch/js/internals/search-insights',
-        'recommend'             : 'Algolia_AlgoliaSearch/js/internals/recommend.min',
-        'recommendJs'           : 'Algolia_AlgoliaSearch/js/internals/recommend-js.min',
-        'rangeSlider'           : 'Algolia_AlgoliaSearch/js/navigation/range-slider-widget',
+        'algoliaBundle'             : 'Algolia_AlgoliaSearch/js/internals/algoliaBundle.min',
+        'algoliaSearchLib'          : 'Algolia_AlgoliaSearch/js/lib/algolia-search.min',
+        'algoliaAutocompleteLib'    : 'Algolia_AlgoliaSearch/js/lib/algolia-autocomplete.min',
+        'algoliaAnalytics'          : 'Algolia_AlgoliaSearch/js/internals/search-insights',
+        'querySuggestionsPluginLib' : 'Algolia_AlgoliaSearch/js/lib/query-suggestions-plugin.min',
```

Comment:
Thinking we should prefix with `algolia` here? 
```suggestion
        'algoliaQuerySuggestionsPluginLib' : 'Algolia_AlgoliaSearch/js/lib/query-suggestions-plugin.min',
```